src/modules/auth/actions.js

```diff
@@ -0,0 +1,13 @@
+import service from './services'
+
+export const authenticateWithGoogle = (socialId, scope) => (
+  (dispatch) => (
+    service.authenticate(socialId, scope)
```

Comment:
We have a different way of working with async actions, we use `redux-promise`. Could you take a look at the boilerplate? it's a lot easier and less code needs to be written.